src/ace/managers/ptplayer.c

```diff
@@ -1231,7 +1232,7 @@ static void mt_playvoice(
 			// Set repeat
 			pChannelData->n_looped = 1;
 			pChannelData->n_replen = pSampleDef->uwRepeatLength;
-			uwSampleLength = pSampleDef->uwRepeatLength + ubFineTune * 2;
+			uwSampleLength = pSampleDef->uwRepeatLength;
```

Comment:
note to self: check against original .s file if it was there  or if it's some kind of bug due to asm->c translation

Reply:
in mt_playvoice there's this fragment

```m68k
	; set repeat
	move.w	(a0),d0
	move.w	d0,n_replen(a2)
	add.w	d3,d0
	add.l	d3,d2
	add.l	d3,d2
	bra	set_loop
```

And earlier on there's calculation of fine tune

```m68k
	moveq	#0,d3
	move.b	(a0)+,d3 ;;; d3 := nibble
	add.w	d3,d3      ;;; nibble *= 2
```

What I overlooked is that there's also between them:

```
	move.w	(a0)+,d3		; repeat offset
```